client/package-lock.json

```diff
@@ -12,6 +12,7 @@
         "@testing-library/react": "^13.4.0",
         "@testing-library/user-event": "^13.5.0",
         "axios": "^1.1.2",
+        "beets": "file:..",
```

Comment:
Do you know why this was added?

Reply:
I was having issues getting this file up to date with the main, and it let me merge any changes you guys made but it didn't let me remove the things that I think you guys deleted. I tried to make sure the files were all the same as before but some might have slipped past me. Let me know what needs to be changed from those files to make them correct and I'll do that before this pull request goes through

Reply:
both package-lock.json files may be wrong

Reply:
Awesome, looks good